File: javascript-closure-angular-client/API/Client/SalesCalculatedTax.js
goog.provide('API.Client.SalesCalculatedTax');

/**
 * @record
 */
API.Client.SalesCalculatedTax = function() {}

/**
 * @type {!API.Client.SalesTaxByType}
 * @export
 */
API.Client.SalesCalculatedTax.prototype.taxByType;

/**
 * Sum of tax type not VAT (Value Added Tax)
 * @type {!number}
 * @export
 */
API.Client.SalesCalculatedTax.prototype.tax;

/**
 * @type {!Array<!API.Client.SalesTaxByTypeDetail>}
 * @export
 */
API.Client.SalesCalculatedTax.prototype.details;

File: async-scala-client/src/main/scala/io/swagger/client/model/SalesCalculatedTax.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class SalesCalculatedTax (
  taxByType: Option[SalesTaxByType],
tax: Option[Double],  // Sum of tax type not VAT (Value Added Tax)
details: Option[List[SalesTaxByTypeDetail]]
)
